docs/README.md

```diff
@@ -1,11 +1,27 @@
-Contributor documentation
+Documentation Field Guide
 =========================
 
 The primary audience for documentation in this folder is contributors to ASP.NET Core.
 If you are looking for documentation on how to *use* ASP.NET Core, go to <https://docs.asp.net>.
 
-# ASP.NET Core developer workflow
+> :bulb: If you're a new contributor looking to set up the repo locally, the [build from source documentation](BuildFromSource.md) is the best place to start.
 
-- [Building from source](BuildFromSource.md)
-- [Troubleshooting build errors](BuildErrors.md)
-- [Artifacts structure](Artifacts.md)
+The table below outlines the different docs in this folder and what they are helpful for.
+
+| Documentation                                                            | What is it about?                                                       | Who is it for?                                                                                                              |
```

Comment:
nit: The extra spaces make this harder to read the source. They also don't change the table layout. Suggest
```suggestion
| Documentation | What is it about? | Who is it for? |
```
and so on for all rows